web-bundle/src/main/java/com/graphhopper/resources/MapboxResource.java

```diff
@@ -105,7 +105,7 @@ public Response doGet(
 
             double minPathPrecision = 1;
             if (overview.equals("full"))
-                minPathPrecision = 0.1;
```

Comment:
Any reason why simplification is disabled? Does it not work or do we avoid a second request for the details?

Reply:
> Any reason why simplification is disabled?

Yes, the `overview` parameter means: "Type of returned overview geometry. Can be  full (the most detailed geometry available),  simplified (a simplified version of the full geometry), or  false (no overview geometry). The default is  simplified". 

I found that the Mapbox SDK works better, when more points are in the geometry. That's why I disabled simplification. Even with disabled simplification, the Mapbox API seems to return a more detailed geometry than GH.

Regarding PathDetails: This is not yet implemented for the Mapbox endpoint. This seems to work  different to the way we do PathDetails and might require a bit of time and planning (haven't checked in detail yet).